alphastats/gui/pages/04_Analysis.py

```diff
@@ -54,21 +53,13 @@
         options=["<select>"]
         + ["------- plots -------"]
         + plotting_options
-        + [
-            key
-            for key in list(get_plotting_options(st.session_state).keys())
-            if key not in plotting_options
-        ]
         + ["------- statistics -------"]
         + list(get_statistic_options(st.session_state).keys()),
     )
 
-    if method in (
-        list((plot_options := get_plotting_options(st.session_state)).keys())
-        + plotting_options
-    ):
+    if method in (plotting_options):
```

Comment:
```suggestion
    if method in plotting_options:
```
is there a hidden reason?

Reply:
nope, just slipped through ;-)